geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommandDUnitTest.java

```diff
@@ -97,47 +93,6 @@ public void createsMappingWithAllOptions() {
       assertThat(mapping.getConnectionConfigName()).isEqualTo("connection");
       assertThat(mapping.getTableName()).isEqualTo("myTable");
       assertThat(mapping.getPdxName()).isEqualTo("myPdxClass");
```

Comment:
It is also removing field-mapping check...I guess we will be removing the support for field mapping check?

Reply:
I think so.

Reply:
we are, but in another pull request.